examples/anchor_text_movie.ipynb

```diff
@@ -17,18 +17,29 @@
   {
```

Comment:
<p>Probably need some comment about the CUDA lines for people without a GPU. Or maybe move these to the language model section and add a sentence? I did run it on a non-GPU machine and it was fine so maybe ok as is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>